packages/time-picker/src/vaadin-time-picker.js

```diff
@@ -166,7 +169,7 @@ class TimePicker extends PatternMixin(InputControlMixin(ThemableMixin(ElementMix
        */
       min: {
         type: String,
-        value: '00:00:00.000',
+        value: '',
```

Comment:
**note:** We can't have a default value here because otherwise, InputConstraintsMixin treats it as an active constraint.